sorc/CMakeLists.txt

```diff
@@ -252,6 +252,7 @@ if (BUILD_UPP)
     "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}"
     "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
     "-Dexec_dir=${CMAKE_INSTALL_BINDIR}"
+#    "-DINTERNAL_IFI=ON"
```

Comment:
@hu5970, this line is commented out. Do we need to keep this line? If not, please remove this line.

Reply:
@chan-hoo I think it is there to uncomment in case the UPP needs to be rebuilt with IFI enabled.

Reply:
I need to have this line here but commented out because I do not want to find this line from UPP building scripts each time when we compile the RRFS with IFI on. It will be uncommented when it goes to operation.

Reply:
Got it.